website/views/shared/avatar/generated_avatar.jade

```diff
@@ -10,6 +10,8 @@ mixin generatedAvatar(options)
   +costumeSetting('back')
   if options.sleep
```

Comment:
Let's be exploit. Have the first option check for true. Also, use triple equals.